app/views/user_mailer/admin_hidden_bookmark_notification.text.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :message do %>
+<%= t("mailer.general.greeting.formal_html", name: @user.login) %>
+
+<%= t(".hidden.text", bookmark_url: bookmark_url(@bookmark)) %>
```

Comment:
The text here should be `Your bookmark of "<%= @bookmark.bookmarkable.title %>" (<%= bookmark_url(@bookmark) %>) has ...` so something similar to what you did for the series.